tests-system/lobster-python/rbt-output-file/default-scenario/input/args.txt

```diff
@@ -0,0 +1,4 @@
+.
```

Comment:
What is the requirement that the test shall verify?
I guess you want to add a test to show the refactoring is working, but the test itself is of minor importance in the context of this refactoring.
Please rename the folder to `rbt-dummy-requirement` in that case, instead of `rbt-output-file`.